xml_inscriptions/transcribed/MI.Detr.DIA.G.25.14.xml

```diff
@@ -272,7 +277,11 @@
    <facsimile>
       <surface>
          <desc/>
-         <graphic url=""/>
+         <graphic url="MI.Detr.DIA.G.25.14.jpg"/>
+      </surface>
+      <surface>
+         <desc/>
+         <graphic url="MI.Detr.DIA.G.25.14 det. .jpg"/>
```

Comment:
We can keep this for now, provided that it works once merged; however, try to avoid spaces/whitespace in the file names for images (and other files).